runtime/src/serde_snapshot/common.rs

```diff
@@ -0,0 +1,168 @@
+use super::*;
+use crate::bank::RefBankFields;
+use std::collections::HashSet;
+use std::sync::RwLock;
+
+#[derive(Default, Clone, PartialEq, Debug, Deserialize, Serialize)]
+pub(crate) struct UnusedAccounts {
+    unused1: HashSet<Pubkey>,
+    unused2: HashSet<Pubkey>,
+    unused3: HashMap<Pubkey, u64>,
+}
+
+// Deserializable version of Bank for snapshot format
+#[derive(Clone, Deserialize)]
+pub(crate) struct DeserializableBankFields {
```

Comment:
Maybe `DeserializableBankFields` and `SerializableBankFields` cannot be unified (maybe unification introduces extra `.clone()`s at periodic snapshot creation)? If so, please add a rationale of it in comment in source code?

Reply:
Commented by my self: https://github.com/solana-labs/solana/pull/10581#issuecomment-652905768